doc/default.nix

```diff
@@ -3,6 +3,10 @@ let
   lib = pkgs.lib;
   sources = lib.sourceFilesBySuffices ./. [".xml"];
   sources-langs = ./languages-frameworks;
+  libDocJSON = builtins.toFile "options.json" (builtins.toJSON (import ./fn-docs.nix));
+  libDocBook= pkgs.runCommand "options-db.xml" {} ''
+    ${pkgs.ruby}/bin/ruby ${./lib2docbook.rb} < ${libDocJSON} > $out
+  '';
```

Comment:
The `fn-docs.nix` -> function docbook conversion is simple enough so it could be done in plain nix instead of using ruby. That will also remove the need of the json conversion.
Nit: `options.json` `options-db.xml` copy paste miss? :)